dspace-spring-rest/src/test/java/org/dspace/app/rest/EPersonRestRepositoryIT.java

```diff
@@ -131,7 +131,7 @@ public void findAllForbiddenTest() throws Exception {
 
         String authToken = getAuthToken(eperson.getEmail(), password);
         getClient(authToken).perform(get("/api/eperson/eperson"))
-                            .andExpect(status().isForbidden());
+                            .andExpect(status().isUnauthorized());
```

Comment:
If you apply the above suggested changes, I think you can also restore the original behavior here.